go.mod

```diff
@@ -72,3 +72,5 @@ require (
 	google.golang.org/grpc v1.31.0 // indirect
 	google.golang.org/protobuf v1.26.0 // indirect
 )
+
+require golang.org/x/sync v0.1.0
```

Comment:
Can you please put this inside the `require` stanza that contains _direct_ dependencies not the one that contains _indirect_ ones (i.e. `// indirect`).

Reply:
Sure, done